Adafruit_AMG88xx.cpp

```diff
@@ -223,40 +221,17 @@ void Adafruit_AMG88xx::read(uint8_t reg, uint8_t *buf, uint8_t num) {
   while (pos < num) {
     buffer[0] = reg + pos;
     i2c_dev->write(buffer, 1);
-#ifdef I2C_DEBUG
-    Serial.print("[$");
-    Serial.print(reg + pos, HEX);
-    Serial.print("] -> ");
-#endif
     uint8_t read_now = min(chuckSize, (uint8_t)(num - pos));
```

Comment:
chunksize could be larger than 255, on some platforms we have 2KB - this will still work if we wrap around - but really it should be a uint32_t or ssize_t or whatever

Reply:
ok, made some changes for that. sry, didn't see you had also approved. want those changes left in?